src/test/testing-library/fixtures/erros.ts

```
@@ -0,0 +1,11 @@
+export const networkMessage: string = 'Failed to fetch';
+export const NETWORK_FAILURE: Error = new Error(networkMessage);
+
+export class ServerError extends Error {
+  public statusCode: number = 500;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)